agoradatatools/etl/transform.py

```diff
@@ -137,6 +151,105 @@ def transform_network(datasets: dict):
 
     return merged
 
+
+def transform_gene_metadata(datasets: dict, adjusted_p_value_threshold, protein_level_threshold):
+    '''
+    This function will perform transformations and incrementally create a dataset called gene_metadata.
+    Each dataset will be left_joined onto gene_info.
+    '''
+    gene_info = datasets['syn25953363']
```

Comment:
One difficulty here is that the code still relies on the Synapse id itself.  Should be refactored

Reply:
I've been thinking of a good way to refactor this and it's not that straight-forward to me.  A custom transformation function needs to be able to reference a particular dataset.  I agree that referencing them by an alias is better, but I'm still tied to a hard-coded reference.